server/app.js

```diff
@@ -12,20 +12,22 @@ app.set('PORT', 3004);
 // serving static files and setting middleware
 app.use(express.static('./public'));
 // sending seed data to client
-app.get('/listinginfo', (request, response) => {
+app.get('/listinginfo/:id', (request, response) => {
+  const listingId = request.params.id;
+  console.log(listingId);
```

Comment:
Using ```request.params``` is a great way to make dynamic endpoints for your CRUD operations. Well done.